app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -823,8 +837,8 @@
       "updCurrUserFormPrefConsErrMsg": "Error updating user form prefs using formID {formId}, and prefs {preferences}: {error}",
       "getCurrUserFormPrefErrMsg": "An error occurred while fetching your preferences for this form.",
       "getCurrUserFormPrefConsErrMsg": "Error getting user form prefs using formID {formId}: {error}",
-      "delCurrformNotiMsg": "Form \"{state.form.name}\" has been deleted successfully.",
-      "delCurrformErrMsg": "An error occurred while attempting to delete \"${state.form.name}\".",
+      "delCurrformNotiMsg": "Form {name} has been deleted successfully.",
+      "delCurrformErrMsg": "An error occurred while attempting to delete {state.form.name}.",
```

Comment:
Hmm, it looks like `delCurrformErrMsg` is never used.

Reply:
It probably needed to be removed and the translation updated. I will remove it